phoenix5-spark/src/main/scala/org/apache/phoenix/spark/datasource/v2/FilterExpressionCompiler.scala

```diff
@@ -15,16 +15,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.phoenix.spark
+package org.apache.phoenix.spark.datasource.v2
```

Comment:
If it is still used in the legacy code, then it may be better not to move.
It is only used by the v2 connectore, then it's fine to move.

Reply:
Generally, do we need the new v2 package in the Spark2 module ?

Reply:
I moved the class back to org.apache.phoenix.spark package.
For v2 package, I think it can be renamed when classes of V1 are removed.

Reply:
IIRC the relevant spark packages also have "v2" in the name, so having similar names in Phoenix makes sense for me.